hypha/static_src/src/sass/apply/components/_filters.scss

```diff
@@ -7,7 +7,7 @@
         right: 0;
         bottom: 0;
         left: 0;
-        z-index: 20;
+        z-index: 9;
```

Comment:
@frjo I have fixed the nav dropdown stacking issue here but just not sure if it is messing with any other UI component.
![image](https://github.com/HyphaApp/hypha/assets/23638629/5583b226-03fc-48a1-abd9-7b4c432b07d0)